packages/astro/src/@types/astro.ts

```diff
@@ -2054,6 +2055,101 @@ export interface AstroUserConfig {
+		 *
+		 * ```js
+		 * ---
+		 * import { PUBLIC_POKEAPI } from "astro:env/client"
+		 * 
+		 * const data = fetch(`${PUBLIC_POKEAPI}/pokemon/squirtle`);
+		 * ---
+		 * To define the data type and properties of your environment variables, declare a schema in your Astro config in `experimental.env.schema`. The `envField` helper allows you define your variable as a string, number, or boolean and pass properties in an object:
+		 * 
+		 * ```js
+		 * // astro.config.mjs
+		 * import { defineConfig, envField } from "astro/config"
+		 * 
+		 * export default defineConfig({
+		 *     experimental: {
+		 *         env: {
+		 *             schema: {
+		 *                 PUBLIC_API_URL: envField.string({ context: "client", access: "public", optional: true }),
+		 *                 PUBLIC_PORT: envField.number({ context: "server", access: "public", default: 4321 }),
+		 *                 API_SECRET: envField.string({ context: "server", access: "secret" }),
+		 *             }
+		 *         }
+		 *     }
+		 * })
+		 * ```
+		 * 
+		 * There are currently 3 data types supported: strings, numbers and booleans.
+		 * 
+		 * There are three kinds of variables, determined by the combination of `context` (`client` or `server`) and `access` (`private` or `public`) settings defined in your [`env.schema`](#experimentalenvschema):
+		 * 
+		 * - **Public client variables**: These variables end up in both your final client and server bundles, and can be accessed from both client and server through the `astro:env/client` module:
+		 * 
+		 *     ```js
+		 *     import { PUBLIC_API_URL } from "astro:env/client"
+		 *     ```
+		 * 
+		 * - **Public server variables**: These variables end up in your final server bundle and can be accessed on the server through the `astro:env/server` module:
+		 * 
+		 *     ```js
+		 *     import { PUBLIC_PORT } from "astro:env/server"
+		 *     ```
+		 * 
+		 * - **Secret server variables**: These variables are not part of your final bundle and can be accessed on the server through the `astro:env/server` module:
+		 * 
+		 *     ```js
+		 *     import { getSecret } from "astro:env/server"
+		 * 
+		 *     const API_SECRET = getSecret("API_SECRET") // typed
+		 *     const SECRET_THAT_MAY_EXIST = getSecret("SECRET_THAT_MAY_EXIST") // string | undefined
+		 *     ```
+		 * 
+		 * **Note:** Secret client variables are not supported because there is no safe way to send this data to the client. Therefore, it is not possible to configure both `context: "client"` and `access: "secret"` in your schema.
+		 * For a complete overview, and to give feedback on this experimental API, see the [Astro Env RFC](https://github.com/withastro/roadmap/blob/feat/astro-env-rfc/proposals/0046-astro-env.md).
```

Comment:
```suggestion
		 * **Note:** Secret client variables are not supported because there is no safe way to send this data to the client. Therefore, it is not possible to configure both `context: "client"` and `access: "secret"` in your schema.
		 *
		 * For a complete overview, and to give feedback on this experimental API, see the [Astro Env RFC](https://github.com/withastro/roadmap/blob/feat/astro-env-rfc/proposals/0046-astro-env.md).
```
Just nicer spacing!